addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/internal/data/Thermostat.java

```diff
@@ -202,7 +222,13 @@ public String getWhereName() {
     }
 
     public Double getAmbientTemperature() {
-        return ambientTemperature;
+        if (getTemperatureUnit() == CELSIUS) {
+            return ambientTemperatureC;
+        } else if (getTemperatureUnit() == FAHRENHEIT) {
+            return ambientTemperatureF;
+        } else {
```

Comment:
Should we log something?

Reply:
I'd rather not add loggers to these simple POJOs that are created for each received update.

When this occurs the channel state will show a null state. So to debug this you would normally want to see what data is received from Nest. Which is already logged in the [NestStreamingRestClient](https://github.com/openhab/openhab2-addons/blob/22550488ac519e34a3e1f7327b33ef5d36047c53/addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/internal/rest/NestStreamingRestClient.java#L183).